README.md

```diff
@@ -103,12 +103,12 @@ spec:
 
 ### Build Strategies Support
 
-| Build Strategy                                                                  | Alpha | Beta | GA |
-| ------------------------------------------------------------------------------- | ----- | ---- | -- |
-| [Source-to-Image](samples/buildstrategy/buildstrategy_source-to-image_cr.yaml)  | ☑     |      |    |
-| [Buildpacks-v3](samples/buildstrategy/buildstrategy_buildpacks-v3-cr.yaml)      | ☑️     |      |    |
-| [Kaniko](samples/buildstrategy/buildstrategy_kaniko_cr.yaml)                    | ☑️     |      |    |
-| [Buildah](samples/buildstrategy/buildstrategy_buildah_cr.yaml)                  | ☑️     |      |    |
+| Build Strategy                                                                                  | Alpha | Beta | GA |
+| ----------------------------------------------------------------------------------------------- | ----- | ---- | -- |
+| [Source-to-Image](samples/buildstrategy/source-to-image/buildstrategy_source-to-image_cr.yaml)  | ☑     |      |    |
```

Comment:
I also found this bug. :)

BTW, should we use the introduction document link instead of the yaml file like (previous like is also just the yaml file link, but it is not good)?
Like: https://github.com/redhat-developer/build/blob/master/docs/buildstrategies.md#buildah

This is a feature introduction: `### Build Strategies Support`
So I think the buildstrategy intro document is better for it.

Reply:
not sure, im just fixing this because is broken and is a bad experience for new people reading the docs. The introduction document is already there see, https://github.com/redhat-developer/build#read-the-docs .

Reply:
Sure thing. If you think this PR is just a fix for the wrong link. That is fine. let us just fix the issue.

Then in future, if we want to improve the link to the document, let us fix laster.